Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt

```diff
@@ -165,13 +167,32 @@ private fun Body(
                         modifier = HzPadding
                     )
                     Spacer(Modifier.height(16.dp))
+                    val seeMore = remember { mutableStateOf(true) }
```

Comment:
Use delegate

```suggestion
                    val seeMore by remember { mutableStateOf(true) }
```

Reply:
If I replace I get an error
![image](https://user-images.githubusercontent.com/16503982/117151159-6b24f200-ad7e-11eb-9274-4499a95f661d.png)

Reply:
You'll also need to add 
```
import androidx.compose.runtime.getValue
import androidx.compose.runtime.setValue
```